sdk/servicebus/service-bus/test/backupMessageSettlement.spec.ts

```diff
@@ -258,7 +258,7 @@ describe("Message settlement After Receiver is Closed - Through ManagementLink",
       ? TestMessage.getSessionSample()
       : TestMessage.getSample();
     const msg = await sendReceiveMsg(testMessages);
-    await receiver.close();
```

Comment:
This test was meant for the back up settlement logic where we need to close the underlying receiver and not the user facing one

Reply:
Why not both?

Reply:
When the user facing receiver is explicitly closed by the user, none of the methods on it should not be usable again. When the settlement methods where on the message this difference did not matter.